src/transformers/modeling_utils.py

```diff
@@ -1231,40 +1274,32 @@ def _generate_no_beam_search(
             outputs = self(**model_inputs)
             next_token_logits = outputs[0][:, -1, :]
 
+            scores = self.postprocess_next_token_scores(
+                next_token_logits,
```

Comment:
can we use keyword arguments here since we have that many inputs to make it more robust:
```python
input_ids=input_ids,
no_repeat_ngram_size=no_repeat_ngram_size,
...
```